frontend/app/src/store/main.ts

```diff
@@ -913,6 +913,16 @@ export class MainStore {
             organization: { ...organization }
           });
         }
+        // Sort based on the "paid" property
+        ps3.sort((a: any, b: any) => {

```

Comment:
Please remove the array sort function, the sort is being done from the database now, thanks

Reply:
Already removed in the second commit